### PR TITLE
Changes from pre-live review on dev

### PR DESCRIPTION
**Pull request by mjones01**
NOTICE: *\* file name change in this PR**
My review noted that the code names were wrong in 03 & 04 (showed 02 & 03 instead), which lead to realizing that the file name for 04 no longer reflected what was actually taught in the lesson.  All names now align with content and are the same across .Rmd, .R, .md and code1.   

Other changes include a few link corrections, spelling error, spaces in comments, and removing tabs from figure titles "main="" ' as these were rending as deltas on the page.

Added proposed combined readme/ contributors so that we'd have an example to look at when discussing new format  (this does NOT transfer to DEV/ live).

### CODE REVIEW

File: 03-when-vector-data-dont-line-up-CRS.Rmd [hunk 1]
```diff
@@ -17,7 +17,7 @@ vector object in R. It will also explore differences in units associated with
 different projections and how to reproject data using spTransform in R. Spatial
 data need to be in the same projection in order to successfully map and process
 them in non-gui tools such as R."
-code1: 02-csv-vector-raster-plotting.R
+code1: 03-when-vector-data-dont-line-up-CRS.R
```
Comment (lwasser) [inline 03-when-vector-data-dont-line-up-CRS.Rmd:20]: awesome. @mjones01 did you test these downloads (download the R file at the bottom of each page)? did they work? i found a few the other day that weren't working (and fixed them) in another repo.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,76 @@
...
+* Introduction to Working With Raster Data in R
+* Working With Raster Time Series Data in R
+* Introduction to Working With Time Series Data in Text Formats in R
+
+## Contributors
+Contributors to this repo include Megan Jones, Sarah Newman, Joseph Stachelek, 
+and Leah Wasser.
+
+## Teaching Data Subsets
+All teaching data subsets can be found on the
```
Comment (lwasser) [inline README.md]: How about something like

Data used in these lessons, can be found on the ...

File: README.md [hunk 3]
```diff
@@ -1,4 +1,76 @@
...
+The results of this collaboration are four tutorial 
+series hosted on the NEON Data Skills portal (neondataskills.org) and also 
+taught as Data Carpentry workshops (www.datacarpentry.org).  
+
+* Introduction to Working With Spatio-Temporal Data and Data Management
+* Introduction to Working With Raster Data in R
+* Working With Raster Time Series Data in R
+* Introduction to Working With Time Series Data in Text Formats in R
+
+## Contributors
```
Comment (lwasser) [inline README.md:42]: we need to think about this language. Sarah is a reviewer whereas joe and you and i both actually developed the content. We should include a contact on this doc too 
Please send questions about these lessons to:
neondataskills@neoninc.org

File: README.md [hunk 4]
```diff
@@ -1,4 +1,76 @@
...
+sources of the data include
+<a href="http://www.neoninc.org/data-resources/get-data/airborne-data" target="_blank"> NEON Airborne Data</a>,
+the <a href="http://landsat.usgs.gov" target="_blank" > U.S. Geological Survey (USGS) Landsat Satellite</a>, 
+the <a href="http://harvardforest.fas.harvard.edu/" target="_blank">Harvard Forest</a>,
+and the <a href="https://www.census.gov/geo/maps-data/data/tiger-cart-boundary.html" target="_blank">US Census Bureau</a> 
+
+---
+
+## Questions?
+Having a problem getting something to work or want to know why we set up
```
Comment (lwasser) [inline README.md:61]: ok ignore the contact statement above :)

File: 03-when-vector-data-dont-line-up-CRS.Rmd (hunk 1 above)
Comment (mjones01) [inline 03-when-vector-data-dont-line-up-CRS.Rmd:20]: All downloaded on DEV (with old name) and in local (with updated name).

File: README.md [hunk 5]
```diff
@@ -1,4 +1,76 @@
...
+something in a certain way? Email us at neondataskills @ neoninc.org, 
+ping Leah Wasser on Twitter 
+[@leahawasser](http://twitter.com/leahawasser) 
+or 
+[file a GitHub Issue](https://github.com/NEONInclwasser/NEON-Data-Skills/issues).
+And if you have a cool resource that you think would add to the site, please let
+us know!
+
+## License
+The main theme of this site was created by Minimal Mistakes. The theme is free
```
Comment (lwasser) [inline README.md]: This site uses the Minimal Mistakes these for Jekyll. The theme is open source software ...

File: README.md (hunk 2 above)
Comment (mjones01) [inline README.md]: I'd rather do the line by line editing in this gDoc (https://docs.google.com/document/d/1yQg0lpcwnAP2pkTFjqLpq2TUChfhpWtZqKL6MsxZfLc/edit?usp=sharing ) and once happy apply changes to all repos.  I added this here so we could see the ReadMe "live" in gitHub as we worked on text in the gDoc.  
I've added your suggestion.

**Comment by lwasser** [comment]
ok - one small change and then i'll merge.

**Comment by mjones01** [comment]
Done

**Comment by mjones01** [comment]
Lessons 01 & 02 used field site to refer both to the new plots added and the full Harvard Forest field site.  Changed the individual sites to "study plots".  Corrected 1 plot title that was incorrect.  Updated others so they also have unambiguous use of field site vs study plot.

**Comment by lwasser** [comment]
Hey Megan - thanks for these edits.
Per our task list, I thought you were done with these lessons and I was on the "push to dev" part. Are you still reviewing these lessons? Let talk in our meeting this afternoon so I'm clear.

**Comment by mjones01** [comment]
I am done with the review for Pre-Dev.  

However, while reviewing the Spatial Intro lessons, which use graphics from Raster (Friday edits) and Vector, I've found things that are either code not working (Raster) or plot titles that were blatantly wrong and in being so challenging to use as the examples in Spatial Intro.  

We needed update plots for the Spatial Intro lessons which meant fixing the Raster/ Vector code.  I don't want to loose those changes so I am pushing up the changes to the other respective repos.

**Comment by lwasser** [comment]
Ok - well communicate this with me please so I don't have to push lessons across multiple times.
Thank you for catching the issues.

**Comment by mjones01** [comment]
Can you please explain how you'd like the information communicated as I felt that this series has been well document?
When I finished the Pre-Dev review, I created a PR and said it was ready to go. You had a few comments so I changed those and commented that they were done. 
As soon as I found issues through other lines of work, I've fixed then pushed them.  Since the PR was still open, they went to it (which I'm assuming means you haven't yet push lessons across), and I've always commented what the changes was for.

**Comment by lwasser** [comment]
I've  been using the google doc task list to keep tabs of things assuming that is most current as you seem to update it as do I. Today I checked the doc and looked at the list. The only thing not crossed off was me accepting rebuilding and pushing.

I went to accept the PR today and saw new edits coming in. let's talk about this today in our meeting. I'd like to be able to see where we are in a process, and know that when something is checked off it is done. And if it isn't done, we revisit it / comment on it.

**Pull request merged**